### PR TITLE
Subpanel removal or swap

### PR DESCRIPTION
**Pull request by kiley-mitti**
Renamed SubPanel to RuxChildContainer. To retain the styles I kept it as a utility component but changed the way it worked to be more RuxContainer-y

Alternately I could just go through to the child containers and add the child-container class to a RuxContainer and that would also work, but this seemed more clean. Let me know!

### CODE REVIEW

**Review by NicDominguez — APPROVED** [review]

File: src/App.css [hunk 1]
```diff
@@ -46,3 +46,37 @@ rux-status {
...
+  font-weight: var(--font-heading-3-font-weight);
+  letter-spacing: var(--font-heading-3-letter-spacing);
+  line-height: var(--font-heading-3-line-height);
+  padding-block: var(--spacing-3);
+  padding-inline: var(--spacing-0);
+}
+
+.child-container::part(body) {
+  gap: var(--spacing-3);
+  box-shadow: 0 0 0 1px inset var(--color-border-interactive-muted);
```
Comment (Paterma) [inline src/App.css:78]: Hooray!
One small thing- this box-shadow can be removed. Based on the PDF it isn't needed (and I know design will be like ....what?), it looks like description should just be` RuxTextarea `.  Most of the default styling on RuxContainer is probably good for most of these containers :) 
Screenshot 2023-07-14 at 11 25 18 AM

**Review by Paterma — CHANGES_REQUESTED** [review]
Just one little change. 
Yay for ripping out these Panels 🎉

**Comment by kiley-mitti** [comment]
so actually now that I look at the mockups this actually needs more work a lot off these don't even need to be child containers. I'm not sure we need child containers at all. working on it a bit. trying to convert to draft but it won't let me. I'll see what I can accomplish Monday!

**Comment by kiley-mitti** [comment]
@Paterma Looking at the wireframes, I removed a bunch of child containers and restyled some of the pages to better match. Then I realized that Frances and I were essentially working in the same areas and so I merged her CSS changes in and resolved conflicts.

There is definitely more cleanup to do.. but this PR is getting large because it's starting to touch every major container. Wondering if we can get it in and then do a bunch of smaller PRs as cleanup.

**Review by Paterma — APPROVED** [review]

File: src/components/ContactDetails/ContactDetails.css [hunk 2]
```diff
@@ -17,17 +18,34 @@
...
+    border-width: 0;
+  }
+  & .Contact-details-grid__equipment-string::part(header) {
+    padding-inline: 0;
+    padding-top: 0;
+  }
+
+  & .Contact-details-grid__equipment-string::part(body),
+  & .Contact-details-grid__equipment-string::part(footer) {
+    box-shadow: 0 0 0 1px inset var(--color-border-interactive-muted);
```
[the remark this answers is not in the document]
Comment (kiley-mitti) [reply src/components/ContactDetails/ContactDetails.css:47]: for SURE. In fact, we may be able to rip out the child-container thing altogether and just use various grids.

**Pull request merged**